ratis-server/src/main/java/org/apache/ratis/server/impl/ServerProtoUtils.java

```diff
@@ -171,4 +171,34 @@ static boolean isSuccess(InstallSnapshotResult result) {
         return false;
     }
   }
+
+  public static String convertToString(InstallSnapshotRequestProto request) {
```

Comment:
Oops, I forgot that we already had `ServerStringUtils.toInstallSnapshotRequestString(..)`.  Filed https://issues.apache.org/jira/browse/RATIS-1676